swagger/v1/swagger.yaml

```diff
@@ -0,0 +1,245 @@
+---
+openapi: 3.0.1
+info:
+  title: API V1
+  version: v1
+components:
+  securitySchemes:
```

Comment:
Hi @Tadessejemal1  and @Kumilachew-g ✋🏼, you have done a great job so far, however, there are a couple of things that you could improve in order to make the project looks more professional, self-explanatory, and clean.

**Please** remove this file since the API documentation is already done and it isn't necessary to have it here.
Note: For working with swagger it is necessary to add the required Gems to the GEM file, so, for the next time it would be just nice to add it in order to make everything work 😃 , as well as add the necessary spec folder with the files to generate the whole skeleton that swagger will consume to generate the API documentation ✋🏼 .